jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -354,7 +357,27 @@ private static DiagnosticProperties getJstatClass(String diagnosticCommand) {
 		bufferPrinter.flush();
 		return DiagnosticProperties.makeStringResult(buffer.toString());
 	}
-	
+
+	@SuppressWarnings("nls")
+	private static DiagnosticProperties loadJVMTIAgent(String diagnosticCommand) {
+		DiagnosticProperties result;
+		String[] parts = diagnosticCommand.split(DIAGNOSTICS_OPTION_SEPARATOR);
+		// parts[0] is already verified as DIAGNOSTICS_LOAD_JVMTI_AGENT since we are here
+		if (parts.length < 2) {
+			result = DiagnosticProperties.makeErrorProperties("Too few arguments, the absolute path of the agent is required : " + diagnosticCommand);
+		} else if (parts.length > 3) {
+			result = DiagnosticProperties.makeErrorProperties("Command not recognized due to more than 3 arguments : " + diagnosticCommand);
+		} else {
+			String attachError = Attachment.loadAgentLibrary(parts[1], (parts.length == 3) ? parts[2] : "", false);
```

Comment:
The test `parts.length == 3` is redundant (we're here because the length is not less than 2 and not greater than 3).

Reply:
Doesn't that mean the length is either 2 or 3. We need to know which case.

Reply:
Yes, you're right. I don't know what I was thinking. :-(